src/doc/rustdoc/src/write-documentation/the-doc-attribute.md

```diff
@@ -230,9 +230,10 @@ If you want to know more about inlining rules, take a look at the
 
 <span id="dochidden"></span>
 
-Any item annotated with `#[doc(hidden)]` will not appear in the documentation, unless
-the `strip-hidden` pass is removed. Re-exported items where one of its ancestors has
-`#[doc(hidden)]` will be considered the same as private.
+Any item annotated with `#[doc(hidden)]` will not appear in the documentation,
+unless the `--document-hidden-items` flag is used.
+Re-exported items where one of its ancestors has
```

Comment:
Hum... This part needs clarification about what "ancestor" is because re-exports of hidden items are displayed.

Reply:
Yeah it sure does! I don't really have access to that information though, I'm working off of very limited documentation.

Reply:
Then please remove this sentence completely.